app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError
+from app.errors import NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    check = True
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+            check = False
+        except NotWearingMaskError:
+            masks_to_buy += 1
+            check = False
+    if masks_to_buy != 0:
```

Comment:
it is okay, but u can just `if masks_to_buy: `, since it will return `True` only if `masks_to_buy` > 0

Reply:
Fixed.